### ISSUE TITLE
Class method can‘t recognize class variable.

### ISSUE THREAD

**Issue by stolenzc** [microsoft/pylance-release#1064]
## Environment data

-   Language Server version: 2021.3.1
-   OS and version: Mac OS 10.15.7
-   Python version (& distribution if applicable, e.g. Anaconda): Python3.8.5

## Expected behaviour

example for my code:
![image](https://user-images.githubusercontent.com/42373706/111577338-07099700-87ed-11eb-8e20-1fa6ad302bfe.png)

I can jump to NAME from cls.NAME, I also can jump to NAME from self.NAME, and all NAME will highlight.

## Actual behaviour

I can't jump to NAME from cls.NAME, bug I can jump to NAME from self.NAME, and cls.NAME not highlight.

```python
class Hello:

    NAME = 'hello'

    @classmethod
    def get_cls_name(cls):
        return cls.NAME

    def get_self_name(self):
        return self.NAME
```

**Comment by stolenzc** [comment]
I update the pylance to 2021.3.2, this problem also exists.

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next release. It occurs specifically when accessing a class member from the `cls` parameter in a classmethod.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2021.3.3, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202133-24-march-2021
